plugins/modules/postgresql_publication.py

```diff
@@ -326,13 +363,15 @@ def check_pub(self):
             # FOR TABLES IN SCHEMA statement is supported since PostgreSQL 15
             if self.pg_srv_ver >= 150000:
                 self.attrs['schemas'] = self.__get_schema_pub_info()
+            if self.pg_srv_ver >= 150000:
+                self.attrs['columns'] = self.__get_columns_pub_info()
```

Comment:
update the comment and lose the duplicate if?
```suggestion
            # FOR TABLES IN SCHEMA statement is supported since PostgreSQL 15, as is publishing by column
            if self.pg_srv_ver >= 150000:
                self.attrs['schemas'] = self.__get_schema_pub_info()
                self.attrs['columns'] = self.__get_columns_pub_info()
```